### ISSUE TITLE
Installation fails in China

### ISSUE THREAD

**Issue by zaoqi** [emscripten-core/emsdk#230]
```
Installing SDK 'sdk-1.38.30-64bit'..
Installing tool 'clang-e1.38.30-64bit'..
Downloading: /home/1828_sandbox/src/the-language/core/arch/emscripten/emsdk/zips/emscripten-llvm-e1.38.30.tar.gz from https://s3.amazonaws.com/mozilla-games/emscripten/packages/llvm/tag/linux_64bit/emscripten-llvm-e1.38.30.tar.gz, 393650696 Bytes
Error downloading URL 'https://s3.amazonaws.com/mozilla-games/emscripten/packages/llvm/tag/linux_64bit/emscripten-llvm-e1.38.30.tar.gz': [Errno 104] Connection reset by peer
Installation failed!
```

**Comment by shogerr** [comment]
Don't worry, it fails in the US too.

**Comment by kripken** [comment]
The emsdk should not be using the old mozilla URLs as of a few days ago, so this may have been fixed. If not, please say what specific emsdk command you are running (perhaps `emsdk install sdk-1.38.30-64bit`? that one works for me now)

**Comment by ligalath** [comment]
I have the same problem when installing in ubuntu16.04 (vmware):
` $./emsdk install 1.38.42-upstream
Installing SDK 'sdk-releases-upstream-05f8c01394ddd0838d3cb484ba8ca97a3efc1ac9-64bit'..
Installing tool 'node-12.9.1-64bit'..
Error: Downloading URL 'https://storage.googleapis.com/webassembly/emscripten-releases-builds/deps/node-v12.9.1-linux-x64.tar.xz': <urlopen error [Errno 104] Connection reset by peer>
Installation failed!
`
emsdk install latest has the same error

**Comment by kripken** [comment]
Github has problems now and then with errors like that, but we're not aware of any at the moment, and our CI which does that all looks green. If this is still a problem for you @ligalath then does just downloading that url in a browser also fail?

**Comment by sbc100** [comment]
Can you use curl of wget to access those URL?  

I suspect that this is not am emsdk but a general connectivty/firewall issue.    I curl and/or wget fail then you need to look into the wider issue and I don't think there is anything we can do on the emsdk side.

**Comment by ligalath** [comment]
> CI has problems now and then with errors like that, but we're not aware of any at the moment, and our CI which does that all looks green. If this is still a problem for you @ligalath then does just downloading that url in a browser also fail?
No, I can't download it from browser either. I thought there is a mechanism in storage.googleapis.com to check my address and refuse me.
I've solved it by setting a proxy.

**Comment by sbc100** [comment]
Great!   Can this issue be closed now?
